dali/dfuplus/dfuplus.cpp

```diff
@@ -64,7 +64,7 @@ class CDafsThread: public Thread
         else
             listenep.getUrlStr(eps);
         enableDafsAuthentication(requireauthenticate);
-        server.setown(createRemoteFileServer());
+        server.setown(createRemoteFileServer()); // no throttle limitting
```

Comment:
Actually will use default throttling I think - is that wise? And if so should probably change the comment.